api/lib/domain/errors.js

```diff
@@ -390,6 +390,12 @@ class CertificationEndedBySupervisorError extends DomainError {
   }
 }
 
+class CertificationEndedByFinalization extends DomainError {
```

Comment:
Pour rester cohérent avec le reste
```suggestion
class CertificationEndedByFinalizationError extends DomainError {
```